testnets/injectivetestnet/assetlist.json

```diff
@@ -30,6 +30,91 @@
         }
       ],
       "type_asset": "sdk.coin"
+    },
+    {
+      "description": "USDC is a USD-pegged stablecoin available on the Injective testnet.",
+      "denom_units": [
+        {
+          "denom": "factory/inj17vytdwqczqz72j65saukplrktd4gyfme5agf6c/usdc",
+          "exponent": 0
+        },
+        {
+          "denom": "USDC",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/inj17vytdwqczqz72j65saukplrktd4gyfme5agf6c/usdc",
+      "name": "USD Coin",
+      "display": "USDC",
+      "symbol": "USDC",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
+      },
+      "coingecko_id": "usd-coin",
```

Comment:
We should define an "additional-mintage" trace to ethereumtestnet::aUSDC
https://github.com/cosmos/chain-registry/blob/3ec283729292ebb8f436f0678a00bce14de5397c/testnets/_non-cosmos/ethereumtestnet/assetlist.json#L97